### ISSUE TITLE
GetPictures cannot retrieve images containing 'alt text'

### ISSUE THREAD

**Issue by shcabin** [qax-os/excelize#2157]
### Description

GetPictures cannot retrieve images containing 'alt text'

### Steps to reproduce the issue

1. insert picture place in cell
2. add Alt Text, and/or "Mark as decorative" 
3. `f.GetPictures("Sheet1", "A1")` return empty

![Image](https://github.com/user-attachments/assets/474d5304-c0b5-4c1c-ac89-cf717206e732)

The generated XML:
add Alt Text: `<rv s="0">
        <v>0</v>
        <v>5</v>
        <v>3D Glasses</v>
    </rv>`
Mark as decorative: `<rv s="0">
        <v>1</v>
        <v>6</v>
        <v xml:space="preserve">Nice Team Sasquatch </v>
    </rv>`

The code implementation requirements are: `len (rv)==2&&rv [1]=="5"`

### Describe the results you received

GetPictures return empty

### Describe the results you expected

GetPictures return  like other picture.

### Go version

1.24.2

### Excelize version or commit ID

2.9.1

### Environment

```shell
Linux
```

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by shcabin** [comment]
I have tried to list some cases 


```
<rv s="0"><v>0</v><v>5</v><v>Smiling alarm clock face</v></rv>
<rv s="0"><v>0</v><v>4</v><v>Smiling alarm clock face</v></rv>
<rv s="1"><v>1</v><v>5</v></rv>
<rv s="1"><v>1</v><v>4</v></rv>
<rv s="0"><v>2</v><v>6</v><v>Bullseye outline</v></rv>
<rv s="0"><v>2</v><v>4</v><v>Bullseye outline</v></rv>
<rv s="2"><v>0</v><v>1</v><v>0</v><v>0</v></rv>
<rv s="2"><v>0</v><v>4</v><v>0</v><v>0</v></rv>
<rv s="3"><v>0</v><v>1</v><v>0</v><v>0</v><v>other alt_text</v></rv>
<rv s="3"><v>0</v><v>4</v><v>0</v><v>0</v><v>other alt_text</v></rv>
<rv s="0"><v>3</v><v>5</v><v>Decorative circles</v></rv>
```
and file: 
[A2.xlsx](https://github.com/user-attachments/files/20760875/A2.xlsx) Created by Microsoft 365, Windows

**Comment by xuri** [comment]
Thanks for your feedback. The `GetPictures` function now support return alt text of cell images, I also fixed missing return some cell images. Please upgrade to the master branch code by `go get -u github.com/xuri/excelize/v2@master`, and this feature will be released in the next version. If you get all cell images in "A2.xlsx" with following code:

```go
package main

import (
    "fmt"

    "github.com/xuri/excelize/v2"
)

func main() {
    f, err := excelize.OpenFile("A2.xlsx")
    if err != nil {
        fmt.Println(err)
        return
    }
    defer func() {
        if err := f.Close(); err != nil {
            fmt.Println(err)
        }
    }()
    cells, err := f.GetPictureCells("Sheet1")
    if err != nil {
        fmt.Println(err)
        return
    }
    for _, cell := range cells {
        pics, err := f.GetPictures("Sheet1", cell)
        if err != nil {
            fmt.Println(err)
            return
        }
        for _, pic := range pics {
            fmt.Printf(
                "cell %s pic's alt: \"%s\", ext: %s, size: %d bytes\r\n",
                cell, pic.Format.AltText, pic.Extension, len(pic.File),
            )
        }
    }
}
```

You will get output like this:

```text
cell B1 pic's alt: "Smiling alarm clock face", ext: .png, size: 63983 bytes
cell D1 pic's alt: "Smiling alarm clock face", ext: .png, size: 63983 bytes
cell B2 pic's alt: "", ext: .png, size: 4519 bytes
cell D2 pic's alt: "", ext: .png, size: 4519 bytes
cell B3 pic's alt: "Bullseye outline", ext: .png, size: 10270 bytes
cell D3 pic's alt: "Bullseye outline", ext: .png, size: 10270 bytes
cell B4 pic's alt: "", ext: .png, size: 4054 bytes
cell D4 pic's alt: "", ext: .png, size: 4054 bytes
cell B5 pic's alt: "other alt_text", ext: .png, size: 4054 bytes
cell D5 pic's alt: "other alt_text", ext: .png, size: 4054 bytes
cell W41 pic's alt: "Decorative circles", ext: .png, size: 50015 bytes
```

**Comment by shcabin** [comment]
rvStruct.getRichDataValueIdx("CalcOrigin") describes the source of the type. Should we define new fields here to separately describe InsertType and Origin?

> Thanks for your feedback. The `GetPictures` function now support return alt text of cell images, I also fixed missing return some cell images. Please upgrade to the master branch code by `go get -u github.com/xuri/excelize/v2@master`, and this feature will be released in the next version. If you get all cell images in "A2.xlsx" with following code:
> 
> package main
> 
> import (
>     "fmt"
> 
>     "github.com/xuri/excelize/v2"
> )
> 
> func main() {
>     f, err := excelize.OpenFile("A2.xlsx")
>     if err != nil {
>         fmt.Println(err)
>         return
>     }
>     defer func() {
>         if err := f.Close(); err != nil {
>             fmt.Println(err)
>         }
>     }()
>     cells, err := f.GetPictureCells("Sheet1")
>     if err != nil {
>         fmt.Println(err)
>         return
>     }
>     for _, cell := range cells {
>         pics, err := f.GetPictures("Sheet1", cell)
>         if err != nil {
>             fmt.Println(err)
>             return
>         }
>         for _, pic := range pics {
>             fmt.Printf(
>                 "cell %s pic's alt: \"%s\", ext: %s, size: %d bytes\r\n",
>                 cell, pic.Format.AltText, pic.Extension, len(pic.File),
>             )
>         }
>     }
> }
> You will get output like this:
> 
> ```
> cell B1 pic's alt: "Smiling alarm clock face", ext: .png, size: 63983 bytes
> cell D1 pic's alt: "Smiling alarm clock face", ext: .png, size: 63983 bytes
> cell B2 pic's alt: "", ext: .png, size: 4519 bytes
> cell D2 pic's alt: "", ext: .png, size: 4519 bytes
> cell B3 pic's alt: "Bullseye outline", ext: .png, size: 10270 bytes
> cell D3 pic's alt: "Bullseye outline", ext: .png, size: 10270 bytes
> cell B4 pic's alt: "", ext: .png, size: 4054 bytes
> cell D4 pic's alt: "", ext: .png, size: 4054 bytes
> cell B5 pic's alt: "other alt_text", ext: .png, size: 4054 bytes
> cell D5 pic's alt: "other alt_text", ext: .png, size: 4054 bytes
> cell W41 pic's alt: "Decorative circles", ext: .png, size: 50015 bytes
> ```


The definition of PictureInsertType seems rather confusing. PictureInsertTypeIMAGE does not appear to belong to the same category as overcell and incell.
In the test case of the document attached here, the output shows that cells B4, B5, D4, D5 are all marked as PictureInsertTypeIMAGE(2). 

However, based on my understanding of the code comments, only the cells using the Image formula (which correspond to B4 and D5) should be assigned the value 2.

**Comment by shcabin** [comment]
rvStruct.getRichDataValueIdx("CalcOrigin") describes the source of the type. Should we define new fields here to separately describe InsertType and Origin?

**Comment by xuri** [comment]
All images added via formula functions are of the inCell type. Among the four values of the `PictureInsertType` enumeration, three of them—everything except `PictureInsertTypePlaceOverCells` are inCell images. Below is an explanation of the origin of each enumeration value:

- `PictureInsertTypePlaceOverCells` represents over cells image added via Insert -> Pictures -> Place Over Cells
- `PictureInsertTypePlaceInCell` represents one cell image added via Insert -> Pictures -> Place In Cell
- `PictureInsertTypeIMAGE` represents one cell image added using the `IMAGE` function
- `PictureInsertTypeDISPIMG` represents one cell image added using the Kingsoft WPS `DISPIMG` function

Therefore, I think needn't introduce a new type to represents category of overCell or inCell images. What do you think about this?

```go
package main

import (
    "fmt"

    "github.com/xuri/excelize/v2"
)

func main() {
    f, err := excelize.OpenFile("A2.xlsx")
    if err != nil {
        fmt.Println(err)
        return
    }
    defer func() {
        if err := f.Close(); err != nil {
            fmt.Println(err)
        }
    }()
    cells, err := f.GetPictureCells("Sheet1")
    if err != nil {
        fmt.Println(err)
        return
    }
    pictureInsertMap := map[excelize.PictureInsertType]string{
        excelize.PictureInsertTypePlaceOverCells: "place over cells",
        excelize.PictureInsertTypePlaceInCell:    "place in cell",
        excelize.PictureInsertTypeIMAGE:          "insert by IMAGE",
        excelize.PictureInsertTypeDISPIMG:        "insert by DISPIMG",
    }
    for _, cell := range cells {
        pics, err := f.GetPictures("Sheet1", cell)
        if err != nil {
            fmt.Println(err)
            return
        }
        for _, pic := range pics {
            fmt.Printf(
                "cell %s pic's %s, alt: \"%s\", ext: %s, size: %d bytes\r\n",
                cell,
                pictureInsertMap[pic.InsertType],
                pic.Format.AltText,
                pic.Extension,
                len(pic.File),
            )
        }
    }
}
```

Output:

```text
cell B1 pic's place in cell, alt: "Smiling alarm clock face", ext: .png, size: 63983 bytes
cell D1 pic's place in cell, alt: "Smiling alarm clock face", ext: .png, size: 63983 bytes
cell B2 pic's place in cell, alt: "", ext: .png, size: 4519 bytes
cell D2 pic's place in cell, alt: "", ext: .png, size: 4519 bytes
cell B3 pic's place in cell, alt: "Bullseye outline", ext: .png, size: 10270 bytes
cell D3 pic's place in cell, alt: "Bullseye outline", ext: .png, size: 10270 bytes
cell B4 pic's insert by IMAGE, alt: "", ext: .png, size: 4054 bytes
cell D4 pic's insert by IMAGE, alt: "", ext: .png, size: 4054 bytes
cell B5 pic's insert by IMAGE, alt: "other alt_text", ext: .png, size: 4054 bytes
cell D5 pic's insert by IMAGE, alt: "other alt_text", ext: .png, size: 4054 bytes
cell W41 pic's place in cell, alt: "Decorative circles", ext: .png, size: 50015 bytes
```
